src/wwwroot/controllers/MainCtrl.js

```diff
@@ -14,11 +14,16 @@
     '$log',
     'ModalService',
     '$route',
-    '$interval'
+    '$interval',
+    '$translate',
+    'utils'
   ];
 
-  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval) {
-    
+  function MainCtrl($rootScope, $scope, $window, $location, auth, $log, ModalService, $route, $interval, $translate, utils) {
+     
+    var key = utils.getPreferredLanguage();
+    $translate.use(key);
```

Comment:
Is this code executed only one time on page loading or on each navigation?

Reply:
It's executed on page loading only, just once.